openlibrary/templates/account/view.html

```diff
@@ -52,7 +52,7 @@ <h1 class="details-title">$header_title</h1>
               $ share_links = ctx.get('share_links')
               $:macros.ShareModal(share_links, page_url, share_content, "share-modal-wrapper", show_embed=False)
         </div>
-      $if page:
+      $if page and mb.is_my_page:
```

Comment:
```suggestion
      $if page:
```
Ensure that `databarView` always appears in the top-right corner of the view.

Reply:
I was totally wrong about this!  The header is too crammed when `databarView` is present for visitors.